test/guppy.tests.js

```diff
@@ -400,6 +401,15 @@ describe('guppy', function () {
       });
     });
 
+    describe('pre-commit gulp options', function () {
+      it('passes options to gulp.src', function () {
```

Comment:
As in my previous comment - `guppy.src` provides list of files that are passed to `gulp.src`, so options could be provided there without altering this library.